example/linux/installation.md

```
@@ -0,0 +1,221 @@
+
+- Globally install `tsx` if not already installed:
+
+```bash
+npm install -g tsx
+```
+
+### 8. Enable and Start the Service
+
+1. Reload the systemd configuration:
+
+
```

Markdownlint (0.37.0)

Expected: 1; Actual: 2
Multiple consecutive blank lines

(MD012, no-multiple-blanks)